src/TodosMain.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { memo } from 'react';
+import { Task } from './Todo';
+import { useTodos } from './hooks/useTodos';
+
+export const TodosMain: React.FC = memo(() => {
+  const { todos } = useTodos();
+
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.map(todo => (
+        <Task todo={todo} key={todo.id} />
```

Comment:
Why you render Task component inside todos array why not Todo component?

Reply:
because of the types, same name everywhere, but now I've renamed type import and everything is ok

Reply:
About the overflow: same stuff on a working example, no? 
<img width="1530" alt="Screenshot 2024-05-17 at 13 32 54" src="https://github.com/mate-academy/react_todo-app/assets/135013788/20b4ea81-c2bf-495a-8dda-eedf415f5402">